.github/workflows/main.yml

```diff
@@ -23,3 +23,26 @@ jobs:
       - name: Run CI tests
         run: npm test
         working-directory: ./tools
+
```

Comment:
Can we make this into a separate job? I think it would be good to keep 'build' strictly for the actual verification/build